tests/infrared/metrics-collectd-qdr.yaml.template

```diff
@@ -4,28 +4,47 @@ tripleo_heat_templates:
 
 custom_templates:
     parameter_defaults:
-        CollectdAmqpInterval: 1
-        CollectdDefaultPollingInterval: 1
-        CollectdExtraPlugins:
+        CollectdAmqpInterval: 5
+        CollectdDefaultPollingInterval: 5
+        CollectdDefaultPlugins:
             - cpu
             - df
+            - disk
             - hugepages
+            - intel_rdt
+            - interface
+            - ipmi
+            - load
+            - memory
             - ovs_events
```

Comment:
ovs should only get installed, when there is ovs running on the node. I'd leave it out for now.
ovs is not the default network in osp anymore since osp15.

Reply:
Oh! That's really important to know, thanks!